OutOfSchool/OutOfSchool.WebApi.Tests/Redis/CacheServiceTests.cs

```diff
@@ -85,14 +86,71 @@ public async Task GetOrAddAsync_WhenDataNotExistsOrExpired_ShouldSaveNewData()
     }
 
     [Test]
-    public async Task RemoveAsync_ShouldCallCacheRemove()
+    public async Task RemoveAsync_ShouldCallCacheRemoveOnce()
     {
         // Arrange & Act
         await cacheService.RemoveAsync("Example");
 
         // Assert
         distributedCacheMock.Verify(
-            c => c.RemoveAsync("Example", It.IsAny<CancellationToken>()),
+            c => c.Remove("Example"),
+            Times.Once);
+    }
+
+    [Test]
+    public async Task ReadAsync_WhenDataExistsInCacheAndNotExpired_ShouldReturnData()
+    {
+        // Arrange
+        var expected = new Dictionary<string, string>()
+        {
+            {"ExpectedKey", "ExpectedValue"},
+        };
+        distributedCacheMock.Setup(c => c.Get(It.IsAny<string>()))
```

Comment:
(Testing wrong data) Please avoid usage of `It.IsAny<>()` whenever possible. Here you know the key thus could specify it in `Setup`. Please do the same in other tests as well

For this specific test it just adds better understanding of the logic. But for some tests it might be crucial.

Reply:
Ok, done. I replaced It.IsAny<> with specific known values.